styles.css

```diff
@@ -0,0 +1,270 @@
+* {
+    Box-sizing: Border-box
+}
+
+html {
+    background-image: url('img/backgroundImage.jpg');
+    background-color: #000000;
+    background-size: 1920px 1080px;
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+    -webkit-font-smoothing: antialiased;
+}
+
+body {
+    font-family: 'Oswald', sans-serif;
+    color: #FFFFFF;
+    padding-top: 1px;
+}
+
+h1  {
+    border-bottom: 2px solid #FFFFFF;
+    padding-bottom: 15px;
+    margin: 220px auto 15px auto;
+    font-size: 5rem;
+    font-weight: 200;
+}
+
+h2  {
+    font-size: 4.2rem;
+    font-weight: 200;
+    margin: 400px auto 20px auto;
+    text-align: center;
+}
+
+h3 {
+    margin: 0 auto 70px auto;
+    font-size: 2.5rem;
+    font-weight: 200;
+}
+
+h2+h3 {
+    text-align: center;
+}
+
+h4 {
+    font-size: 2rem;
+    font-weight: 200;
+    margin: 5px 0 30px 0;
+}
+
+p, li {
+    font-family: 'Raleway', sans-serif;
+    font-size: 1.3rem;
+}
+
+a, a:visited, a:hover, a:active {
+    color: #FFFFFF;
+    text-decoration: none;
+}
+
+header  {
+    width: 50%;
+    position: fixed;
+}
+
+main {
+    width: 50%;
+    position: relative;
+    left: 50%;
+}
+
+section {
+    border: 2px solid #FFFFFF;
+    padding: 0 10px;
+    position: relative;
+    margin-bottom: 40px;
+}
+
+section::before {
+    content: ' ';
+    background-color: #d9d9d9;
+    height: 70px;
+    width: 150px;
+    position: absolute;
+    top: -20px;
+    left: -30px;
+    opacity: 0.5;
+    z-index: -1;
+}
+
+article {
+    border: 2px solid #FFFFFF;
+    padding: 30px;
+    margin-bottom: 100px;
+    -o-transition:.5s;
+    -ms-transition:.5s;
+    -moz-transition:.5s;
+    -webkit-transition:.5s;
+}
+
+article:hover {
+    -webkit-box-shadow: 0px 0px 53px 5px rgba(0,0,0,1);
+    -moz-box-shadow: 0px 0px 53px 5px rgba(0,0,0,1);
+    box-shadow: 0px 0px 53px 5px rgba(0,0,0,1);
+    background-color: rgba(217,217,217,0.3);
+    cursor: pointer;
+}
+
+article h3 {
+    text-align: center;
+    margin-bottom: 30px;
+}
+
+article img {
+    width: 100%;
+}
+
+article p {
+    margin-bottom: 0;
+}
+
+ul {
+    padding: 0;
+    text-align: center;
+}
+
+li {
+    display: inline-block;
```

Comment:
tip: fine this time but try and get in the habit of using float, loads easier and doesn't come with built in bugs

Reply:
Thanks, will do. Am still scared of float.